addons/website_slides_survey/controllers/main.py

```diff
@@ -0,0 +1,60 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo.http import request
+from odoo.osv import expression
+from odoo.addons.website_profile.controllers.main import WebsiteProfile
+
+
+class WebsiteSlidesSurvey(WebsiteProfile):
+    def _prepare_user_profile_values(self, user, **kwargs):
+        """Loads all data required to display the certification attempts of the given user"""
+        values = super(WebsiteSlidesSurvey, self)._prepare_user_profile_values(user, **kwargs)
+        values['show_certification_tab'] = ('user' in values) and (
+            values['user'].id == request.env.user.id or \
+            request.env.user.has_group('survey.group_survey_manager')
+        )
+
+        if not values['show_certification_tab']:
+            return values
+
+        domain = [('survey_id.certification', '=', True)]
+        if values['user'].mapped('email'):
+            domain = expression.AND([domain, ['|',
+                ('email', 'in', values['user'].mapped('email')),
+                ('partner_id', 'in', values['user'].mapped('partner_id.id'))
```

Comment:
Again, mapped usage seems unnecessary. 
Unless there can really be multiple users in "user"? That would seem strange.